api/cryptocom/rest_action_test.go

```diff
@@ -0,0 +1,260 @@
+package cryptocom
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/bmizerany/assert"
+)
+
+type mockHTTPClientError struct {
```

Comment:
Is it ok to use this for test error in Post and Get

Reply:
In general - ok